server/routes/overview/overviewView.ts

```diff
@@ -103,6 +107,24 @@ export default class OverviewView {
       prisonerHasHadInduction && !this.actionPlanReviews?.problemRetrievingData && mostRecentReviewSession == null
     const prisonerHasHadInductionAndAtLeastOneReview = prisonerHasHadInduction && mostRecentReviewSession != null
 
+    const reviewDateFrom = this.actionPlanReviews?.latestReviewSchedule?.reviewDateFrom || null
+    const reviewDateTo = this.actionPlanReviews?.latestReviewSchedule?.reviewDateTo || null
+    const noReviewsDue = this.actionPlanReviews?.latestReviewSchedule?.status === ActionPlanReviewStatusValue.COMPLETED
+    const releaseDate = this.prisonerSummary.releaseDate || null
+
+    const currentDate = new Date()
```

Comment:
I would call this `today` and init as with date-fns:
```
const today = startOfToday()
```

Then, in your logic, I wonder if using some of the date-fns functions such as [isBefore](https://date-fns.org/v4.1.0/docs/isBefore) (and/or isAfter etc) might make the code more readable?